hbase-server/src/test/java/org/apache/hadoop/hbase/backup/TestHFileArchiving.java

```diff
@@ -526,6 +526,56 @@ public void testCleaningRace() throws Exception {
     }
   }
 
+  @Test
+  public void testArchiveRegionTableAndRegionDirsNull() throws IOException {
+    Path regionDir = new Path(FSUtils.getTableDir(new Path("./"),
+            TableName.valueOf(name.getMethodName())), "abcdef");
+    Path familyDir = new Path(regionDir, "cf");
+    Path rootDir = UTIL.getDataTestDirOnTestFS("testCleaningRace");
+    Path file = new Path(familyDir, "0");
+    Path sourceFile = new Path(rootDir, file);
+    FileSystem fileSystem = UTIL.getTestFileSystem();
+    fileSystem.createNewFile(sourceFile);
```

Comment:
We are almost there. This line is not needed, and neither are lines #531 to 533, and 535, since we are not doing anything with familyDir, regionDir and sourceFile in this test. Basically all we need here is the the fileSystem and rootDir itself.

Reply:
This makes sense. Updating the PR